.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
This file is just for informing to git to ignore some IDE specific files (like here the .idea directory created by IntelliJ IDEA)